index.html

```diff
@@ -212,293 +234,583 @@ <h2>Dependencies</h2>
+    <section>
+      <h2><code>NEL</code> response header</h2>
 
-          <dt>tls.cert.authority_invalid</dt>
-          <dd>The TLS connection was aborted due to invalid issuing authority</dd>
+      <p>
+      The <dfn><code>NEL</code></dfn> <a>response header</a> is used to
+      communicate an <a>origin</a>'s <a>NEL policy</a> to the user agent.  The
+      ABNF (Augmented Backus-Naur Form) syntax for the <a>NEL</a> header is as
+      follows:
+      </p>
 
-          <dt>tls.cert.invalid</dt>
-          <dd>The TLS connection was aborted due to invalid certificate</dd>
+      <pre>NEL = json-field-value</pre>
 
-          <dt>tls.cert.revoked</dt>
-          <dd>The TLS connection was aborted due to revoked server certificate</dd>
+      <p>
+      The header's value is interpreted as an array of JSON objects, as defined
+      by <a>json-field-value</a>. Each object in the array defines an <a>NEL
+        policy</a> for the origin. The user agent MUST process the first valid
+      policy in the array and ignore any additional policies in the array.
+      </p>
 
-          <dt>tls.cert.pinned_key_not_in_cert_chain</dt>
-          <dd>The TLS connection was aborted due to a key pinning error</dd>
+      <p>User agents MUST ignore any unknown or invalid field(s) or value(s) that do not conform to the syntax defined in this specification. A valid <a>NEL</a> header field MUST, at a minimum, contain one object with all of the "REQUIRED" fields defined in this specification.</p>
 
-          <dt>tls.protocol.error</dt>
-          <dd>The TLS connection was aborted due to a TLS protocol error</dd>
+      <p>The user agent MUST ignore the <a>NEL</a> header specified via a <code>meta</code> element to mitigate hijacking of error reporting via scripting attacks. The <a>NEL policy</a> MUST be delivered via the <a>NEL</a> <a>response header</a>.</p>
 
-          <dt>tls.failed</dt>
-          <dd>The TLS connection failed due to reasons not covered by previous errors</dd>
+      <p class="note">The restriction on <code>meta</code> element is consistent with the [[CSP]] specification, which restricts reporting registration to HTTP header fields only for the same reasons.</p>
 
-        </dl>
-        <dl class='reportTypeGroup'>
-          <dt>http.protocol.error</dt>
-          <dd>The connection was aborted due to an HTTP protocol error</dd>
+      <section>
+        <h2>The <code>report_to</code> member</h2>
 
-          <dt>http.response.invalid</dt>
-          <dd>Response is empty, has a content-length mismatch, has improper encoding, and/or other conditions that prevent user agent from processing the response</dd>
+        <p>
+        The <dfn><code>report_to</code></dfn> member specifies the <a>endpoint
+          group</a> that reports for this <a>NEL policy</a> will be sent to.
+        The <a>report_to</a> member is REQUIRED to register a <a>NEL policy</a>,

```

Comment:
Is the REQUIRED necessary here? In reporting we have a [fallback to "default"](https://wicg.github.io/reporting/#id-member), any reason why we shouldn't extend the same behavior here?

Reply:
I like that!  I'd like to keep this as a refactoring PR, so I opened #84 to make sure to change this in a follow-on PR.

Reply:
👍